Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -88,16 +88,14 @@ extension HTTPClient {
 
     /// Represent HTTP request.
     public struct Request {
-        /// Request HTTP version, defaults to `HTTP/1.1`.
-        public var version: HTTPVersion
         /// Request HTTP method, defaults to `GET`.
-        public var method: HTTPMethod
+        public let method: HTTPMethod
```

Comment:
does method have to be `let`? It's not part of the url change verification, so we can keep it as `var` (maybe move closer to all other vars in the file?